db/listener_test.cc

```diff
@@ -687,6 +687,7 @@ class TableFileCreationListener : public EventListener {
   class TestEnv : public EnvWrapper {
    public:
     explicit TestEnv(Env* t) : EnvWrapper(t) {}
+    const char* Name() const override { return "TestEnv"; }
```

Comment:
why we do not have static const char* kClassName() here?

Reply:
See above.